frontend/packages/core/src/Resolver/index.tsx

```diff
@@ -132,6 +138,7 @@ const Resolver: React.FC<ResolverProps> = ({
               <SchemaResolver key={schema.typeUrl} schema={schema} submitHandler={submitHandler} />
             ))}
           </AccordionGroup>
+          <NotePanel notes={notes} />
```

Comment:
should this be placed above Query Resolver? Saw the screenshot in the internal PR & the notes get placed at the bottom which may be missed by the user.

Reply:
yeah I wasn't sure where to put it. I'll move it to the top